src/blockchains/cardano/cardano_worker.ts

```diff
@@ -5,9 +5,6 @@ import util from './lib/util';
 import { logger } from '../../lib/logger';
 
 
-const CARDANO_GRAPHQL_URL = process.env.CARDANO_GRAPHQL_URL || 'http://localhost:3100/graphql';
```

Comment:
Move those variables to where the others live. They come in the constructor argument.